activerecord/test/models/company.rb

```diff
@@ -26,6 +26,9 @@ def arbitrary_method
   def private_method
     "I am Jack's innermost fears and aspirations"
   end
+
+  class SpecialCo < Company
+  end
```

Comment:
Perhaps there is another class `Firm` that is outside of the `Namespaced` module?